apps/web/src/components/PageEditor.tsx

```diff
@@ -89,10 +102,13 @@ export function PageEditor({ page, onSave, isSaving }: PageEditorProps) {
       <AuthDialog open={!isAuthenticated} />
       <div className="mb-4 flex flex-row justify-between">
         <input
+          ref={inputRef}
```

Comment:
✨ You can actually specify `autofocus` as an attribute!

Reply:
Yeah, I did that first, and it didn't work for some reason.  So ended up using the `inputRef` instead